esmvaltool/utils/cmorizers/obs/cmorize_obs_MTE.py

```diff
@@ -0,0 +1,60 @@
+"""ESMValTool CMORizer for MTE data.
+
+Tier
+    Tier 2: other freely-available dataset.
```

Comment:
This is actually Tier 3 since a registration is required.

Reply:
But what kind of observations belong to Tier 2? For MTE a registration is required, but everyone can get an account for free. I thought this is the definition of Tier 2.

Reply:
Whenever a registration is required we put that under Tier 3, this is because usually it implies the acceptance of some kind of licence/conditions.